content/faq/errors.md

```diff
@@ -41,3 +41,7 @@ Scope [<module_import_chain>]
 ```
 
 Circular dependencies can arise from both providers depending on each other, or typescript files depending on each other for constants, such as exporting constants from a module file and importing them in a service file. In the latter case, it is advised to create a separate file for your constants. In the former case, please follow the guide on circular dependencies and make sure that both the modules **and** the providers are marked with `forwardRef`.
+
+#### "Sequqlize & Sequlelize-typescript compatibilty issue" error
+
+If you are using Sequqlize-typescript and installed version of sequelize is 6.6.5 then you might see an error while defining your model. To solve this, install sequelize version 6.6.2
```

Comment:
Could we add a small sample of what that error might look like? Maybe something like
```bash
node_modules/sequelize-typescript/dist/model/model/model.d.ts:10:31 - error TS2417: Class static side 'typeof import("<projectRoot>/node_modules/sequelize-typescript/dist/model/model/model").Model' incorrectly extends base class static side 'typeof import("<projectRoot>/node_modules/sequelize/types/lib/model").Model'.
```

Reply:
I have added the sample error and some more text to it